alphadia/workflow/manager.py

```diff
@@ -494,12 +494,16 @@ def __init__(
                 "fwhm_mobility": config["optimization_manager"]["fwhm_mobility"],
                 "score_cutoff": config["optimization_manager"]["score_cutoff"],
             }
-            self.__dict__.update(initial_parameters)
+            self.__dict__.update(
+                initial_parameters
+            )  # TODO either store this as a dict or in individual instance variables
 
             for key, value in initial_parameters.items():
                 self.reporter.log_string(f"initial parameter: {key} = {value}")
 
-    def fit(self, update_dict):
+    def fit(
+        self, update_dict
+    ):  # TODO siblings' implementations have different signatures
```

Comment:
yes.. this is unfortunately necessary :/ Should we name the functions differently in this case?

Reply:
this would be a solution, yes .. otherwise using inheritance does not make too much sense (but maybe it does not in the first place?)